src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -192,6 +213,13 @@ object ErrorAggregator {
       stats("gmplugin_crashes") = ping.getCountKeyedHistogramValue("SUBPROCESS_CRASHES_WITH_DUMP", "gmplugin")
       stats("content_shutdown_crashes") = ping.getCountKeyedHistogramValue("SUBPROCESS_KILL_HARD", "ShutDownKill")
 
+      for {
+        (histogramName, (processTypes, thresholds)) <- thresholdHistograms
+        processType <- processTypes
+        threshold <- thresholds
+      } stats(s"${histogramName}_${processType}_${threshold}") =
```

Comment:
Maybe move this to a function since it's used more than once? (the create name portion)